node-swc/src/binding.js

```diff
@@ -5,9 +5,19 @@ const { platform, arch } = process
 
 let nativeBinding = null
 let localFileExisted = false
-let isMusl = false
 let loadError = null
 
+const isMusl = () => {
+  try {
+    const isMusl = readFileSync('/usr/bin/ldd', 'utf8').includes('musl')
```

Comment:
I mean this file is needlessly loaded as a node js string.

Reply:
I see. tho it's worth mentioning this PR doesn't change that

I don't mind changing to something else, what did you have in mind?

Reply:
Oh, I see.
Sorry and then it's fine.